src/box/errcode.h

```diff
@@ -319,10 +319,10 @@ struct errcode_record {
 	/*264 */_(ER_NIL_UUID,			"Nil UUID is reserved and can't be used in replication") \
 	/*265 */_(ER_WRONG_FUNCTION_OPTIONS,	"Wrong function options: %s") \
 	/*266 */_(ER_MISSING_SYSTEM_SPACES,	"Snapshot has no system spaces") \
-	/*267 */_(ER_CLUSTER_NAME_MISMATCH,	"Cluster name mismatch: expected %s, got %s") \
-	/*268 */_(ER_REPLICASET_NAME_MISMATCH,	"Replicaset name mismatch: expected %s, got %s") \
+	/*267 */_(ER_CLUSTER_NAME_MISMATCH,	"Cluster name mismatch: name '%s' provided in config confilcts with the instance one '%s'") \
```

Comment:
> NO_CHANGELOG=internal change
> NO_DOC=internal change

I agree that it doesn't require a documentation request, but this is a user visible change, not internal one.

So, I would say that the patch doesn't change the public API (an error message is not a part of the API) and so shouldn't be documented.

Regarding changelog: there was no release with these error messages, so there is no sense to mention the intermediate change in the release notes. The same point here: I agree to don't add the changelog entry, but the reason is different.

Reply:
updated